ci/setup-new-node/enable-buildkite.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
"setup-new-node/" -- "node" is an overloaded term.  ""setup-new-buildkite-agent/" ?